src/emqx.erl

```diff
@@ -290,6 +284,53 @@ on_shutdown(_) ->
 %%--------------------------------------------------------------------
 %% Internal functions
 %%--------------------------------------------------------------------
+-ifdef(EMQX_ENTERPRISE).
+applications_need_restart() ->
+    [gproc, esockd, ranch, cowboy, ekka, emqx].
+-else.
+applications_need_restart() ->
+    [gproc, esockd, ranch, cowboy, ekka, emqx, emqx_modules].
+-endif.
+
+-define(PK_START_APPS, {?MODULE, default_started_applications}).
+default_started_applications() ->
+    case persistent_term:get(?PK_START_APPS, undefined) of
+        undefined ->
+            AppNames = expand_apps(applications_need_restart()),
```

Comment:
what's the value today after expansion ?

Reply:
For emqx opensource: 
```
[gproc,ecpool,ehttpc,esockd,lwm2m_coap,gen_coap,ranch,
 cowboy,grpc,minirest,ekka,emqx,emqx_modules]
```
Enterprise:
```
[gproc,ecpool,ehttpc,esockd,lwm2m_coap,gen_coap,ranch,
 rabbit_common,cowboy,grpc,minirest,ekka,emqx]
```